src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -2,20 +2,370 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.Globalization;
 using Microsoft.Xunit.Performance;
+using Xunit;
 
 namespace System.Tests
 {
     public class Perf_Double
     {
         public static readonly double dValue = 1.23456789E+5;
+        private const int iterCount = 20000;
 
-        [Benchmark]
-        public static void Double_ToString()
+        [Benchmark(InnerIterationCount = iterCount)]
+        [InlineData(104234.343)]
+        [InlineData(double.MinValue / 2)]
+        [InlineData(Math.PI)]
+        [InlineData(Math.E)]
+        [InlineData(double.MaxValue)]
+        [InlineData(double.MinValue)]
+        [InlineData(double.NaN)]
+        [InlineData(double.PositiveInfinity)]
+        [InlineData(double.NegativeInfinity)]
+        [InlineData(0xFFFFFFFFFFFFF)]
```

Comment:
Sorry, I was on mobile and tried to keep it short. I meant that you have to do `double d = BitConverter.Int64BitsToDouble(unchecked((long)(0x8000000000000000)));`

Attempting to print this will give `0` (rather than `-0.0`) due to https://github.com/dotnet/coreclr/issues/10290

In either case using `[InlineData(-0.0)]` will work as expected:
```C#
double d = -0.0;
Console.WriteLine($"0x{BitConverter.DoubleToInt64Bits(d):X}");
// Prints: 0x8000000000000000
```